lambda/create-investigation/handler.py

```
@@ -155,12 +211,17 @@ def lambda_handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
             if TASK_TIMEOUT_MINIMUM > 0 and task_timeout < TASK_TIMEOUT_MINIMUM:
                 raise ValueError(f"task_timeout below minimum ({TASK_TIMEOUT_MINIMUM}s)")
         except (ValueError, TypeError) as e:
-            logger.warning(f"Invalid task_timeout: {task_timeout} — {str(e)}")
+            logger.warning(f"Invalid task_timeout: {task_timeout} — {e!s}")
+            if TASK_TIMEOUT_MINIMUM > 0:
+                return response(400, {
+                    'error': f'task_timeout must be between {TASK_TIMEOUT_MINIMUM} and 86400'
+                })
             return response(400, {
-                'error': f'task_timeout must be an integer between {TASK_TIMEOUT_MINIMUM} and 86400'
+                'error': f'task_timeout must be between 0 and 86400'
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)